neo.UnitTests/UT_ProtocolHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Akka.TestKit.Xunit2;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.Network.P2P;
+using Neo.Network.P2P.Capabilities;
+using Neo.Network.P2P.Payloads;
+using System;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_ProtocolHandler : TestKit
+    {
+        private static readonly Random TestRandom = new Random(1337); // use fixed seed for guaranteed determinism
+
+        private NeoSystem testBlockchain;
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+            Shutdown();
+        }
+
+        [TestInitialize]
+        public void TestSetup()
+        {
+            Akka.Actor.ActorSystem system = Sys;
+            var config = TestKit.DefaultConfig;
+            var akkaSettings = new Akka.Actor.Settings(system, config);
+            testBlockchain = TestBlockchain.InitializeMockNeoSystem();
+        }
+
+        [TestMethod]
+        public void ProtocolHandler_Test_SendVersion_TellParent()
+        {
+            var senderProbe = CreateTestProbe();
+            var parent = CreateTestProbe();
+            var protocolActor = ActorOfAsTestActorRef<ProtocolHandler>(() => new ProtocolHandler(testBlockchain, parent));
```

Comment:
```suggestion
            var protocolActor = parent.ActorOfAsTestActorRef(() => new ProtocolHandler(testBlockchain));
```

Try this?

Reply:
Looks like this works Erik. It allows us to have a single parameter on ProtocolHandler.